bookkeeper-server/src/main/java/org/apache/bookkeeper/replication/Auditor.java

```diff
@@ -141,7 +141,7 @@ private void initialize(ServerConfiguration conf, ZooKeeper zkc)
             LedgerManagerFactory ledgerManagerFactory = AbstractZkLedgerManagerFactory
                     .newLedgerManagerFactory(
                         conf,
-                        bkc.getRegClient().getLayoutManager());
+                        bkc.getMetadataClientDriver().getLayoutManager());
```

Comment:
not for this patch, but ledgermanagerfactories are created from the metadata driver. Shouldn't we do the same here?

Reply:
that is in subsequent change. all the changes I am dealing here is about bookkeeper client and reg client. I will have a subsequent change for bookie and reg manager, a change for ledger manager and a change (or few changes) for auditor & underreplicated managers.

Reply:
:+1: